.github/workflows/build_ubuntu.yml

```diff
@@ -229,6 +223,49 @@ jobs:
           name: user-guide
           path: ${{ steps.create-user-guide.outputs.pdf-path }}
 
+  test_assets:
+    runs-on: ubuntu-latest
```

Comment:
duplicated? there is another Pr with the same purpose

Reply:
Yes it will disapear when merging and updating from develop. Should have set this one as draft meanwhile